### PR TITLE
fix: error messages are not shown in setup

### PR DESCRIPTION
**Pull request by ayZagen**
vee validate scopes are used but they are missing in errors parameter

### CODE REVIEW

File: dev/webpack/webpack.dev.js [hunk 1]
```diff
@@ -35,6 +35,7 @@ module.exports = {
     pathinfo: true,
     crossOriginLoading: 'use-credentials'
   },
+  devtool: 'cheap-module-eval-source-map',
```
Comment (NGPixel) [inline dev/webpack/webpack.dev.js]: Submit a different PR for this and make it optional, as it slows down the build.

Comment (karolherbst) [reply dev/webpack/webpack.dev.js]: maybe just merge the commit without that change? 21 days is a bit long for such a trivial change and I was running into this issue as well and though the setup just wasn't working at all...

**Comment by ayZagen** [comment]
sorry, I was on a vacation. I will resolve this as soon as I return back to my computer :)
